app/views/layouts/angular-bootstrap/_endpoints_angular.html.haml

```diff
@@ -143,7 +153,8 @@
 
 %div{"ng-if" => defined?(tls_ca_certs_hide) ? false : true}
   .form-group{"ng-if"=> "emsCommonModel.emstype == 'rhevm' || " +                                          |
-                        "(emsCommonModel.ems_controller == 'ems_container' && " +                          |
+                        "((emsCommonModel.ems_controller == 'ems_container' || " +                         |
+                        "emsCommonModel.emstype == 'hawkular') && " +                               |
```

Comment:
Grouping them different seems a good option that would give more flexibility to modify them as needed for each provider without having to copy/paste the whole block.